Core/PARStore.m

```diff
@@ -768,7 +774,7 @@ - (NSData *)dataFromPropertyList:(id)plist error:(NSError **)error
 
 - (id)propertyListFromData:(NSData *)blob error:(NSError **)error
 {
-    if (!blob)
+    if (!blob || blob.length == 0)
```

Comment:
I guess one thing we may have lost here is the ability to store an empty data, right? Will now always become nil. Does raise the question of whether some specific data value for null might not be better. Eg. some string converted to data.

Reply:
We decided it was fine to use an empty data as placeholder as a real empty NSData property list value would be still encoded as a plist and therefore correctly discriminated from the placeholder

Reply:
Addressed